pylint_odoo/checkers/modules_odoo.py

```diff
@@ -351,6 +357,14 @@ def visit_import(self, node):
             if isinstance(node.scope(), astroid.Module):
                 self._check_imported_packages(node, name)
 
+    @utils.check_messages('except-pass')
+    def visit_tryexcept(self, node):
+        """Visit block try except"""
+        for handler in node.handlers:
```

Comment:
@moylop260 @lasley 
Done, Now we exclude if the exception is assigned to one variable